package.json

```diff
@@ -212,7 +213,7 @@
     "react-native-aes-crypto": "rainbow-me/react-native-aes#65c49f7e70266615b2999eaa7db654d3fe4f2e3b",
     "react-native-ble-plx": "2.0.3",
     "react-native-bootsplash": "4.3.2",
-    "react-native-branch": "5.3.1",
+    "react-native-branch": "5.9.0",
```

Comment:
gotta revert this bump, not working on android

Reply:
oops not sure how that got in